### PR TITLE
M3 fetching data

### PR DESCRIPTION
**Pull request by blakbox23**
In this part of the project I : 
- Connected the project to an API
- Stored items to the API
- Retrieved items from the API
- Displayed items according to the API

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,27 +1,74 @@
...
-    case REMOVE_BOOK:
+    case GET_BOOK:
+      return action.formattedData;
 
-      return state.filter((book) => book.id !== action.payload.id);
+    case REMOVE_BOOK:
+      return state.filter((book) => book.item_id !== action.payload.item_id);
 
     default:
       return state;
```
Comment (Emmyn5600) [inline src/redux/books/books.js]: - [x] Good Job, data are being sent and stored using API, however, let us modify our implementation a little bit, to keep our code clean and organized, let us create a new file called API.js to keep `(POST, DELETE and GET)` and book.js  file to keep the `action types, creator and reducers.`

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @blakbox23 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Data are being sent and received to/from API 

rr25

     Changes required :recycle: 
    
    -  Please check the comment under the review too

    Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @blakbox23 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.

 #### Optional suggestions
 
 -N/A   

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by blakbox23** [comment]
Thanks @Emmyn5600 👍🏾

**Pull request merged**